invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesCarousel/index.js

```diff
@@ -5,31 +5,67 @@
  * Invenio is free software; you can redistribute it and/or modify it
  * under the terms of the MIT License; see LICENSE file for more details.
  */
-import React from "react";
-import ReactDOM from "react-dom";
-import CommunitiesCarousel from "./CommunitiesCarousel";
-import { OverridableContext, overrideStore } from "react-overridable";
+import $ from "jquery";
 
 const communitiesCarouselContainer = document.getElementById("communities-carousel");
-const title = communitiesCarouselContainer.dataset.title;
-const fetchUrl = communitiesCarouselContainer.dataset.fetchUrl;
 const intervalDelay = parseInt(communitiesCarouselContainer.dataset.intervalDelay);
 const animationSpeed = parseInt(communitiesCarouselContainer.dataset.animationSpeed);
-const defaultLogo = communitiesCarouselContainer.dataset.defaultLogo;
-const itemsPerPage = communitiesCarouselContainer.dataset.itemsPerPage;
-
-const overriddenComponents = overrideStore.getAll();
-
-ReactDOM.render(
-  <OverridableContext.Provider value={overriddenComponents}>
-    <CommunitiesCarousel
-      title={title}
-      fetchUrl={fetchUrl}
-      intervalDelay={intervalDelay}
-      animationSpeed={animationSpeed}
-      defaultLogo={defaultLogo}
-      itemsPerPage={itemsPerPage}
-    />
-  </OverridableContext.Provider>,
-  communitiesCarouselContainer
-);
+
+const carouselSlides = $(".carousel.transition");
```

Comment:
would it be possible to use `#` instead of `.`? Otherwise, if `.carousel.transition` is a child of an id, `$("#id").find(".carousel.transition")` is much more performant. (I used jQuery here, but we should do it without.